context-free.md

```diff
@@ -1,22 +1,26 @@
 # Context Free
 
 ##  Who is this programming language for?
+It seems to be intended for anyone who wants to create computer art with it. From looking at the gallery, I think there are quite a few math enthusiasts among the user base.
```

Comment:
I like the comment about math enthusiasts. Think about the features of the language. Do you think they shape the intended user and the art those users create?